app/controllers/hq/sponsors_controller.rb

```diff
@@ -1,6 +1,6 @@
 class Hq::SponsorsController < HqController
   def index
-    @sponsors= Sponsor.all
+    @sponsors = Sponsor.paginate(:page => params[:page])
```

Comment:
Notice: I didn't use `Sponsor.all.paginate(:page => params[:page])` like in the Partner#index controller. They both result in the same SQL query.